website/static/js/filepage/index.js

```diff
@@ -363,6 +364,37 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+            var editable = self.editor && self.editor.selected;
```

Comment:
Essentially. Use history.js to modify the query parameters without doing a page refresh. If our components are wired correctly, simply changing that query parameter should enable the revisions view.